### ISSUE TITLE
Sequence example shows diagonal lines

### ISSUE THREAD

**Issue by pericazivkovic** [TLmaK0/gravizo#16]
Sequence example rendering broken just like in example on Gravizo.com
![2016-10-07-203345_385x371_scrot](https://cloud.githubusercontent.com/assets/535911/19201310/722c7df6-8ccd-11e6-90a2-13ff451c7f14.png)

**Comment by TLmaK0** [comment]
Thanks @pericazivkovic, now it's working, I will investigate it.

**Comment by pericazivkovic** [comment]
Yep, it is ok for me too in the last two-three days. Feel free to close this if you wish

**Comment by plgregoire** [comment]
I think the problem is back

![image](https://cloud.githubusercontent.com/assets/3228187/19398589/61072a64-921b-11e6-8ef6-99de5f6934cb.png)

**Comment by TLmaK0** [comment]
@pericazivkovic @plgregoire I was unable to catch it. If you see it again, please, use the svg verstion to check if it returns the same problem:  ````http://g.gravizo.com/svg?``` instead of ```http://g.gravizo.com/g?```.

**Comment by pericazivkovic** [comment]
@TLmaK0 issue is back and only for g.gravizo.com/g? not for SVG url

![g](https://cloud.githubusercontent.com/assets/535911/20227494/cdc61c36-a84c-11e6-9f70-5db365e14356.png)

**Comment by TLmaK0** [comment]
@pericazivkovic Thanks, I will continue investiganting it. This will help a lot.

**Comment by TLmaK0** [comment]
Fixed

**Comment by TLmaK0** [comment]
Not fixed
